introduction/from_flat_php_to_symfony2.rst

```diff
@@ -370,7 +370,7 @@ on the requested URI::
         echo '<html><body><h1>Page Not Found</h1></body></html>';
     }
 
-For organization, both controllers (formerly ``index.php`` and ``show.php``)
+For organization, both controllers (formerly ``/index.php`` and ``/index.php/show``)
```

Comment:
I agree with you, so the filename above should be changed accordingly, that's what I meant :).